### PR TITLE
Workaround back-level powershell get-content -> string.object[] probl…

### PR DESCRIPTION
**Pull request by petemounce**
…em on reading.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: scripts/packages/chocolatey/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -2,16 +2,14 @@ $ErrorActionPreference = 'Stop'; # stop on all errors
 $packageName = 'bazel'
 
 $toolsDir = Split-Path -parent $MyInvocation.MyCommand.Definition
-$paramsText = get-content "$($toolsDir)\params.txt"
+$paramsText = ((get-content "$($toolsDir)\params.txt" | out-string) -split "`n")
 write-host "Content of $($toolsDir)\params.txt:"
 write-host $paramsText
```
Comment (laszlocsomor) [inline scripts/packages/chocolatey/tools/chocolateyinstall.ps1]: This is rendered incorrectly:

```
PS C:\Users\laszlocsomor> echo foo bar baz > a.txt ; type a.txt
foo
bar
baz
```

```
PS C:\Users\laszlocsomor> $foo = ((get-content "a.txt" | out-string) -split "`n")
PS C:\Users\laszlocsomor> write-host $($foo.GetType())
System.String[]
PS C:\Users\laszlocsomor> write-host $foo
 baz
PS C:\Users\laszlocsomor> write-host $foo[0]
foo
PS C:\Users\laszlocsomor> write-host $foo[1]
bar
PS C:\Users\laszlocsomor> write-host $foo[2]
baz
```

Maybe print the results of get-content (the object array) before converting it to a string array?

Comment (petemounce) [inline scripts/packages/chocolatey/tools/chocolateyinstall.ps1]: Done; good spot!

**Comment by iirina** [comment]
@laszlocsomor can you merge this?

**Comment by laszlocsomor** [comment]
@iirina : I'll import this now.

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**